src/services/paras/ParasService.ts

```diff
@@ -45,51 +71,44 @@ export class ParasService extends AbstractService {
 		return {
 			at,
 			fundInfo,
+			// TOOD would it make snese to merge the below derived info into `fundInfo`
+			// or put them under another key like `fundInfoDerive`?
+			leasePeriods,
 		};
 	}
 
 	/**
-	 * @param hash
-	 * @param includeFundInfo
-	 * @returns
-	 * {
-	 *   "at": {}
-	 *   "funds": [
-	 *     {
-	 *       "paraId": number,
-	 *       "fundInfo": {}
-	 *     }
-	 *   ]
-	 * }
+	 * @param hash `BlockHash` to make call at
+	 * @param includeFundInfo wether or not to include `FundInfo` for every crowdloan
+	 * @returns list of all crowdloans
```

Comment:
Same as before. Maybe we could remove these `@returns` and write doc info above the `@param` notes.

Reply:
I don't think this is really that important more so an observation of continuity between all services.